### ISSUE TITLE
Mixer: size issues (workaround)

### ISSUE THREAD

**Issue by joliclic** [Jojo-Schmitz/MuseScore#669]
I'm under Linux, Ubuntu 24.04, my screen size is 1920x1200. 
Elements in the Mixer view are really too big.
The workaround is to launch the appimage with some QT values :

`env QT_AUTO_SCREEN_SCALE_FACTOR=0 QT_SCALE_FACTOR=1 QT_SCREEN_SCALE_FACTORS=1 MuseScore-3.7-latest.AppImage -w %F`

The size of elements in the Start Center are big too (without the workaround), that's ugly but usuable. The size of the splashscreen, when the app starts, seems better too with the correction.

Note: the workaround can be used in a launcher, i.e. a .desktop file, in the Exec= value.

**Comment by joliclic** [comment]
![Musescore mixer too big](https://github.com/user-attachments/assets/fcae3c5d-588d-41f3-84f8-df26d23c049d)
screenshot of the mixer too big

**Comment by Jojo-Schmitz** [comment]
And this is different from 3.6.2?

**Comment by joliclic** [comment]
I have the same problem with 3.6.2

**Comment by Jojo-Schmitz** [comment]
OK, so not a regression. What about MuseScore 4?

**Comment by Jojo-Schmitz** [comment]
Not isn't an issue on my Windows 11, with the same resolution (and a 100% scaling). Isa this a HighDPI screen (higher than ~100DPI), i.e. what is the physical size (it is a 24" screen here)?

**Comment by Jojo-Schmitz** [comment]
So it __is__ HighDPI, Sqrt(1920² + 1200²) / 14" ~ 162 DPI
MuseScore is known to have issues with that

**Comment by Jojo-Schmitz** [comment]
Not sure it exists on Linux, but in Windows there's a scaling value, on my (low DPI) screen set to 100%, but on a 14" laptop easily set to 160%. If you have such a setting, try reducing to like 120%

**Comment by joliclic** [comment]
You means by apps ? I have such an available setting in Gnome, but this modify the whole desktop, so all apps...
But my workaround works very well, this bug doesn't bother me anymore ;) . Hope this can help somebody else.
Thanks for maintaining this version 3.7!

**Comment by Jojo-Schmitz** [comment]
Yep, on Windows that setting too affects everthing, not just a single app.

**Comment by Jojo-Schmitz** [comment]
Does https://musescore.org/en/node/233416 help?

**Comment by joliclic** [comment]
That's funny, the -D argument change the size of the UI elements, but not the mixer!
UI Font size doesn't change too, but should be normal because of the settings in the preferences.

**Comment by Jojo-Schmitz** [comment]
There's also a -x option, a stretch factor

**Comment by joliclic** [comment]
the -x option change the icons size, but like the -D the size of the mixer controls does not change for me, nor the start center
